model/event/version_event.go

```diff
@@ -67,3 +70,24 @@ func LogVersionGithubCheckFinishedEvent(id, newStatus string) {
 		}))
 	}
 }
+
+func LogVersionChildrenCompletionEvent(id, status, author string) {
```

Comment:
Seems as though this and `LogPatchChildrenCompletionEvent` could share a helper function that passes in ResourceType, EventType as parameters